### ISSUE TITLE
fork/pair down koans

### ISSUE THREAD

**Issue by brian-dawn** [clojurebridge-minneapolis/installfest#80]
After running through the koans a couple times myself and a couple TAs were discussing cutting out symbols/quotes/lists/sets and some of the redundant koans.

This would leave us more time to spend on other T2 projects.

**Comment by crkoehnen** [comment]
Ted and I were talking with a student during lunch and she had questions
about symbols and quotes.  It might be that the koans, on their own, are
insufficient.  I think symbols and quote are particularly important and
would prefer adding a ten minute prepared talk (along the lines of the git
intro) to dropping them.

Lists and sets I'm more ambivalent about.  They're straight forward, well
documented, and behave more or less like lists and sets in other languages.

**Comment by brian-dawn** [comment]
The koans are definitely insufficient on their own in a lot of ways - what's nice is that we usually have productive conversations as a group when students hit these situations. We had discussed symbols as a group but I think they are fundamentally weird and not really necessary for a lot of Clojure code. To be honest I don't even use them at all even when writing macros.

I think you're right that a well prepared talk is preferable to removing them completely.

The Koans are great for teaching syntax but I think it might be better if the students got to work on a full program, in which case the koans as they are now are less than ideal as we quickly run out of time.

How does this sound, a 10 minute intro on Clojure and we discuss homoiconicity and symbols/quoting, then we remove some unnecessary sections from the Koans, try to get through everything in the morning then in the afternoon the students can try to work on bigger projects?

**Comment by elenam** [comment]
Last time I've seen Track 2 koans was a couple of years ago, so I am not sure what the state of it is now. There was a companion write-up with it back then that I wrote a part of that's been subsequently edited and improved by various people. Do you still have it? Perhaps a write-up on quotes and lists would be a better approach than a talk since different people get to it at various times. 
It would be great to have pointers from koans to the corresponding parts of the write-ups and clojuredocs.

**Comment by elenam** [comment]
I would be happy to work on it since I would love to revisit Track 2 materials.

**Comment by brian-dawn** [comment]
@elenam yeah I ported it over to the new docs here: http://clojurebridge-minneapolis.github.io/clojure_functional.html unfortunately we don't mention quoting in there much and I also noticed that students weren't following along with it, they were just doing the koans.

Maybe that was a failure on my part to make it clear there was an accompanying document D: but I also think students just started churning through the koans since a lot of them are solvable without any supporting docs.

If you want to do a writeup on quoting/homoiconicity that would be amazing! I struggled to learn that stuff as well when I was learning Clojure and I'm still unsure how best to teach it to others.

**Comment by brian-dawn** [comment]
After looking through that doc there's a lot of good stuff there, we should maybe pull some things out into separate docs/talks.

It's just hard to find a balance, I really like that the koans let the students immediately start writing Clojure but there are some things that might be better as more of a lecture.

**Comment by brian-dawn** [comment]
So student feedback here, some of the feedback goes along with this tickets goals which I think is great... it means we can iterate and improve our next one! https://github.com/clojurebridge-minneapolis/organizing/blob/master/Fall-2016/survey.md
